### ISSUE TITLE
Subprocess returncode invalid-unary-operand-type false positive when psutil is present

### ISSUE THREAD

**Issue by VFLashM** [PyCQA/pylint#1431]
### Steps to reproduce

Make sure `psutil` is installed.

Create `test.py` with following content:
```py
import subprocess
import psutil
process = subprocess.Popen(['ls'])
process.communicate()
process_signal = -process.returncode
```

### Current behavior
```
pylint -E test.py
************* Module test
E:  5,17: bad operand type for unary -: NoneType (invalid-unary-operand-type)
E:  5,17: bad operand type for unary -: NoneType (invalid-unary-operand-type)
E:  5,17: bad operand type for unary -: NoneType (invalid-unary-operand-type)
E:  5,17: bad operand type for unary -: NoneType (invalid-unary-operand-type)
```

If `import psutil` is commented, pylint reports no errors.

### Expected behavior

No errors regardless of `psutil`.

### pylint --version output
```
pylint 1.7.1, 
astroid 1.5.2
Python 2.7.12 (default, Sep 20 2016, 14:04:58) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-17)]
```

**Comment by Pierre-Sassoulas** [comment]
Closing as the problem is fixed in 2.14 or I cannot reproduce.
